near-sdk-sim/Cargo.toml

```diff
@@ -1,22 +1,21 @@
 [package]
 name = "near-sdk-sim"
-version = "2.0.0"
+version = "3.0.0-pre-release"
```

Comment:
I am not sure how "-pre-release" version work; I know for sure that "-alpha.N" and "-beta.N" (e.g. "-alpha.1", "-beta.7") work fine

Reply:
@matklad do you have a suggestion or thought on this one?

Reply:
This one is still waiting on a release from the node-runtime crate so we can fix this when actually trying to do a prerelease.

Reply:
the convention is I think `-pre.1` or `-beta.1`. It has affordance for `-pre.2` if you do need to make changes. 

Generally, it is `-anysuffix`, sever doesn't assign any particular meaning there.